src/Polly.Strategies/Events.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace Polly;
+
+public delegate ValueTask EventsDelegate<T, TContext>(Outcome<T> outcome, TContext context);
+
+public class Events<TContext>
```

Comment:
What's the purpose of this type in the big picture?

Reply:
I have updated the docs and added [more details](https://github.com/App-vNext/Polly/blob/mtomka/v8-poc/docs/Polly-V8.md#handling-of-different-result-types) about these.